app/components/user_list/index.tsx

```diff
@@ -127,6 +140,8 @@ export default function UserList({
     handleSelectProfile,
     fetchMore,
     loading,
+    manageMode = false,
+    showManageMode = false,
```

Comment:
what is the difference between this two? can they be combined?

Reply:
The default behavior for the `UserListRow` component (`manageMode = false`) shows the `check-circle` icon for selecting and unselecting a user.  `manageMode=true` lets the component know the default icon will not be used.  Instead, a `chevron-down` icon is shown (depending on the `showManageMode` prop).

#### **default icon `manageMode=false`**
<img width="396" alt="image" src="https://user-images.githubusercontent.com/7575921/204699217-9f074873-d7ce-4ede-944d-c38befff60ed.png">

#### **`manageMode=true`  && `showManageMode=false`**
<img width="405" alt="image" src="https://user-images.githubusercontent.com/7575921/204698946-56a87084-1469-401b-b838-5b0dba221b55.png">


#### **`manageMode=true`  && `showManageMode=true`**
<img width="403" alt="image" src="https://user-images.githubusercontent.com/7575921/204698979-5199cb66-38ff-4f3a-b20f-a793ed090e53.png">